apps/landing/.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:vue/recommended",
+        "plugin:@typescript-eslint/recommended",
+        "@vue/eslint-config-typescript"
+    ],
+    "parser": "vue-eslint-parser",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "parser": "@typescript-eslint/parser",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "vue",
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "max-len": ["error", { "code": 200, "ignoreStrings": true, "comments":300, "ignoreTemplateLiterals": true }],
+        "vue/multi-word-component-names": "off",
+    },
+    "ignorePatterns": [
+        "**/node_modules/**",
+        "**/build/**",
+        "**/dist/**",
+        "**/lib/**",
+        "cdk.out"
+    ]
+}
```

Comment:
Can you remove the parcel cache files right below this entry? I don't think these will be generated again either, but might be worth checking.

Reply:
I thought we just got rid of those files altogether because we weren't using them anymore. I don't see parcel anywhere in our project anymore. And if you still want me to take an action, are you saying to add those potential build files to the `ignorePatterns` array?

Reply:
Oh! Looks like they were actually removed and I was squinting too hard.

Reply:
Don't squint so hard!